assets/src/edit-story/elements/square/save.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import getPercentageFromPixels from '../../utils/getPercentageFromPixels';
+
+/**
+ * Returns AMP HTML for saving into post content for displaying in the FE.
+ */
+function SquareSave( { id, backgroundColor, width, height, x, y, rotationAngle } ) {
+	const style = {
+		position: 'absolute',
```

Comment:
Same, not the focus here, but still this is similar to our convo on wrappers and the repeated code that supports the element placement/sizing/rotation basics.

Reply:
Improved in 5469f09.